### PR TITLE
Add drop shadow again on icons in the topbar too

### PR DESCRIPTION
**Pull request by nickvergessen**
### Before
Bildschirmfoto von 2020-05-14 13-34-54

### After
Bildschirmfoto von 2020-05-14 13-34-12

It's at least a bit better, but the `…` is very small so the shadow is also small

### CODE REVIEW

File: css/icons.scss [hunk 1]
```diff
@@ -46,11 +46,14 @@
...
 		.app-navigation-toggle,
 		.action-item__menutoggle {
-			color: #fff;
+			color: #ffffff;
+			filter: drop-shadow(1px 1px 4px var(--color-box-shadow));
 		}
 	}
 
 	.forced-white {
+		filter: drop-shadow(1px 1px 4px var(--color-box-shadow));
```
Comment (jancborchardt) [inline css/icons.scss:55]: We don’t need both the circle _and_ the drop-shadow, right? We can remove the drop-shadows I’d say

Comment (nickvergessen) [reply css/icons.scss:55]: This also affects the other icons which don't have a circle around them.
But we can have a look at it again when this is live

Comment (jancborchardt) [reply css/icons.scss:55]: Yeah, no big deal :) Looks nicer if the icons with the circle don’t have the drop-shadow, just something for the long list cc @ma12-co

**Pull request merged**